webapp/channels/src/components/post_view/post_list_virtualized/post_list_virtualized.tsx

```diff
@@ -720,6 +720,7 @@ export default class PostList extends React.PureComponent<Props, State> {
                                             ref={this.listRef}
                                             height={height}
                                             width={width}
+                                            id='post-list-virtualized'
```

Comment:
Nitpicking the ID a bit here, but could you change it to `postListScrollContainer`? I think that name is a bit more self-explanatory for when we're trying to debug this component in the future, and that name matches its parent's `postListContent` ID which is used in a bunch of E2E tests